It should not error on pulumi warnings

For example, it should not error when a new version of pulumi is available.

```
##[error]warning: A new version of Pulumi is available. To upgrade from version '1.6.1' to '1.7.0', visit https://pulumi.com/docs/reference/install/ for manual instructions and release notes.
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -1,9 +1,6 @@
 name: Test
 
-on:
-  push:
-    branches:    
-      - master
+on: push
 
 jobs:
   test:

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -12,7 +12,7 @@ jobs:
     - uses: prepor/setup-pulumi@master
       with:
         version: 1.3.4
-    - uses: .
+    - uses: ./
       with:
         stack: dev
         args: up

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -12,7 +12,7 @@ jobs:
     - uses: prepor/setup-pulumi@master
       with:
         version: 1.3.4
-    - uses: prepor/github-pulumi@master
+    - uses: .
       with:
         stack: dev
         args: up

--- a/src/index.ts
+++ b/src/index.ts
@@ -56,7 +56,7 @@ async function run() {
                 core.info(s)
             },
             stderr: (data: Buffer) => {
-                core.error(data.toString())
+                core.warning(data.toString())
             }
         },
         ignoreReturnCode: true

